released/packages/coq-bits/coq-bits.1.1.0/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+maintainer: "anton.a.trunov@gmail.com"
+
+homepage: "https://github.com/coq-community/bits"
+dev-repo: "git+https://github.com/coq-community/bits.git"
+bug-reports: "https://github.com/coq-community/bits/issues"
+license: "Apache-2.0"
+
+synopsis: "A bit vector library"
+description: """
+A formalization of bitset operations in Coq and the corresponding
+axiomatization and extraction to OCaml native integers"""
+
+build: [make "-j%{jobs}%"]
+install: [make "install"]
+depends: [
+  "ocaml" {(>= "4.05" & < "4.13~")}
```

Comment:
```suggestion
```
From the GitHub repo, it looks like the default `make` build for this project doesn't use any OCaml at all, just pure Coq. So I would advise either removing the OCaml bound or the whole OCaml constraint entirely.

Reply:
To verify axioms, there is a Makefile target `verify`, which uses `patch`, `ocamlbuild` and `ocaml`. I guess it's better to keep the `ocaml` dependency (I can remove the constraint) and add `ocamlbuild` as a dependency. What do you think?

Reply:
If you actually run the `verify` task as part of the opam building or installation process, then it would make sense to include both `ocamlbuild` and `ocaml` as dependencies. But the build commands are only running and installing the pure Coq code, so I still think you should remove the `ocaml` dependency.

Reply:
My recommendation for future releases is that you turn axiom verification into a part of the *testing* process of the opam package. Then you can have `ocamlbuild` as a dependency qualified by `with-test`, and people who only want to use the package without testing it can ignore it.

Reply:
Applied your suggestion in e6f0c26. Thanks!

Reply:
Oh, wait, does CI use `--with-test`? In that case, the 16-bit axiom verification is going to time out.

Reply:
CI doesn't use `--with-test` to my knowledge.

Reply:
It does:
https://github.com/coq/opam-coq-archive/blob/ac66d366523e171a8c89a62614816ea5cd61cf5b/scripts/opam-coq-install-remove#L76